crates/starknet-devnet-types/src/rpc/transactions/broadcasted_declare_transaction_v1.rs

```diff
@@ -72,11 +73,19 @@ impl BroadcastedDeclareTransactionV1 {
         let class_info: ClassInfo =
             ContractClass::Cairo0(self.contract_class.clone()).try_into()?;
 
-        Ok(DeclareTransaction::new(
-            sn_api_declare,
-            starknet_api::transaction::TransactionHash(transaction_hash.into()),
-            class_info,
-        )?)
+        if FieldElement::from(self.common.version) > QUERY_VERSION_OFFSET {
```

Comment:
Should it be `>` or `>=`? Also applicable to other occurrences of comparison with `QUERY_VERSION_OFFSET`. In other words, what about transactions with version 0 (i.e. their query counterparts)?

Reply:
I questioned myself for it, but checked out the transactions versions and each transaction starts at version 1